packages/core/src/Stores/notification-store.js

```diff
@@ -205,7 +206,11 @@ export default class NotificationStore extends BaseStore {
             id,
             buy_price,
             contract_id,
-            contract_type: getTradeTypeName(contract_type, isHighLow({ shortcode }), isTurbosContract(contract_type)),
```

Comment:
@kate-deriv Do we expect to get `' Accumulators'` with a space (when `getContractBasis(contract_type)` returns an empty string)?

Reply:
You're right, sorry
Will add `trim()`